packages/components/src/components/Modal/index.tsx

```diff
@@ -199,6 +209,22 @@ const CancelIconWrapper = styled.div`
     cursor: pointer;
 `;
 
+const ProgressBarPlaceholder = styled.div<{ hiddenProgressBar: boolean }>`
+    height: 4px;
+    width: 100%;
+    margin-bottom: 20px;
+    background-color: ${props => (props.hiddenProgressBar ? 'white' : colors.NEUE_STROKE_GREY)};
+`;
+
+const GreenBar = styled.div<{ width: number }>`
+    height: 4px;
+    position: relative;
+    background-color: ${colors.GREEN};
+    z-index: 3;
```

Comment:
just a nitpick, this zindex is most likely not needed (or I just still don't fully understand stacking context in css) 😀